src/bci_build/package/templates/licensing_and_eula.j2

```diff
@@ -0,0 +1,22 @@
+## License
+SPDX-License-Identifier: {{ image.license }}
+
+{%if image.is_opensuse %}This image is based on [openSUSE Tumbleweed](https://get.opensuse.org/tumbleweed/).
+{% else -%}
+{% if image.support_level|string == "techpreview" -%}
+This image is a tech preview. Do not use it for production use.
```

Comment:
Duplication of "use":
```suggestion
This image is a tech preview, it is not intended for production use.
```

Reply:
changed it to "Do not use it for production"